tests/tensorflow/test_extensions.py

```diff
@@ -12,18 +12,30 @@
 import transformer_engine_tensorflow as tex
 import tensorflow as tf
 
+from tensorflow.python.eager import context
 from tensorflow.python.framework import test_util
 from tensorflow.python.platform import test
 
 from transformer_engine.tensorflow import TE_DType
 from transformer_engine.tensorflow import get_stream_id
 
+def skip_test():
+    if not tf.test.is_gpu_available(True, (9, 0)):
+        return 'Fp8 requires Hopper+ GPU'
+    if len(context.context().list_physical_devices('GPU')) != 1:
+        return 'Only supports a single GPU'
+    return None
```

Comment:
This skips the non-fp8 tests on <9.0 GPUs. Maybe something like:
```suggestion
def skip_test(need_fp8=False):
    if need_fp8:
        if not tf.test.is_gpu_available(True, (9, 0)):
            return 'Fp8 requires Hopper+ GPU'
    if len(context.context().list_physical_devices('GPU')) != 1:
        return 'Only supports a single GPU'
    return None
```
This would need to be called in each test so we can set `need_fp8` if needed.